### ISSUE TITLE
Question: Is it possible to combien Pulid + control net?

### ISSUE THREAD

**Issue by ClothingAI** [ToTheBeginning/PuLID#130]
Title.
For even more precision.

**Comment by ToTheBeginning** [comment]
Yes, it is possible.

**Comment by LongVu219** [comment]
> Yes, it is possible.

Do we need to retrain the model ? If yes, how can we do that, i can't find the training code.

**Comment by LongVu219** [comment]
one more question, i want to modify your loss function and some of your network architecture, is it possible to do that without the training code ?
